tests/configs/benchmark_config_spec_test.py

```diff
@@ -38,6 +38,10 @@
 _GCP_AWS_DISK_CONFIG = {'GCP': {}, 'AWS': {}}
 
 
+def _GetFlagDict(flag_values):
+  return {name: flag_values[name] for name in flag_values}
```

Comment:
The flags.FLAGS object is a dict-like object, the tests is comparing a dictionary with _GetFlagDict().